lib/source-map-consumer.js

```diff
@@ -14,7 +14,7 @@ var quickSort = require('./quick-sort').quickSort;
 function SourceMapConsumer(aSourceMap) {
   var sourceMap = aSourceMap;
   if (typeof aSourceMap === 'string') {
-    sourceMap = JSON.parse(aSourceMap.replace(/^\)\]\}'/, ''));
```

Comment:
note: if you reuse the same regexp (see below), please note that only the parens needs escaping.

Reply:
I did do it this way fwiw.